pravega/pravega
# Issue 6803: Fix standalone log location prompt

**Change log description**  
Correct the log location prompt.

**Purpose of the change**
Fixes #6803.

**How to verify it**
Launch Pravega in standalone mode, and check the `standalone.log` and the prompt.
